.github/workflows/2x-image-build-test.yaml

```diff
@@ -0,0 +1,25 @@
+name: 2.x Image Build
+
+on:
+  pull_request:
+    branches:
+      - '*'
+  push:
+    branches:
+      - 'main'
+      - 'next'
+    tags:
+      - '2.*'
+      - 'v2.*'
+
+jobs:
+  verify-docker-build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Verify Railgun Docker Build
+      uses: docker/build-push-action@v2
+      with:
+        push: false
+        file: Dockerfile.railgun
+        tags: latest
+        cache-from: type=local,src=/tmp/.buildx-cache
```

Comment:
Won't this cache be empty because `actions/cache` is not appropriately executed?

Reply:
yep, 9b3acd7c3c5fa02fbe5562b532b3b7c4dc396e7c should correct for this.